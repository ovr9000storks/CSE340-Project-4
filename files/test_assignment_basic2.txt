
a , b  ;

{
a = 457 ;
b = 221 ; 
input a;
output a ;
output b ;
input b;
a = 65537 ; 
b = 12481632 ; 
output a ;
output b ;
}
9 18 2
