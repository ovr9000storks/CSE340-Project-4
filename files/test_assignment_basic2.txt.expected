9 221 65537 12481632 